crates/serpen/tests/fixtures/ast_rewriting/function_name_collision/main.py

```
@@ -0,0 +1,24 @@
+#!/usr/bin/env python3
+"""
+Function name collision test.
+Both modules define a function with the same name.
+"""
+
+from module_a import process_data as process_a
+from module_b import process_data as process_b
+
+
+def main():
```

Ruff (0.11.9)

Missing return type annotation for public function `main`

(ANN201)